meerkat_frontend/src/js/technical/tabletSubmission.js

```diff
@@ -211,51 +178,52 @@ function fillTabletData(clinicObj) {
     });
 }
 
-//This method will handle all the data related to week columns "tot and reg"
-function buildWeekData(clinicObj, minuWeek, colKey) {
-    var weekVal = week - minuWeek;
-    $.each(clinicObj.deviceSubmissions, function(index, deviceVal) {
-        $.each(tabletDataArray, function(index, value) {
-            if (value.DevicesIDs === "- DeviceId  " + deviceVal.deviceId) {
-                var itemName = colKey + weekVal;
-                value[itemName] = deviceVal.submissionsCount;
-            }
-        });
-    });
-}
 
+//this method will handle to fill the data from the apis into the table
+function fillAPIResultData(clinicObj, minusNum, colKey) {
```

Comment:
Why not simply changing the function name to `fillAPIResultDataIntoTable` and get rid of that comment?

Reply:
edited